.editorconfig

```diff
@@ -14,6 +17,15 @@ block_comment_start = /*
 block_comment = *
 block_comment_end = */
 
+[*.json]
```

Comment:
minor but according to the doc you can group rules with :
```
[*.{js,json,yaml,yml}]
indent_size = 2
```

Reply:
Thanks :-)